package.json

```diff
@@ -44,42 +44,47 @@
     "@react-navigation/stack": "^5.14.4",
     "array-flat-polyfill": "^1.0.1",
     "axios": "^0.21.1",
-    "babel-jest": "^26.6.3",
+    "babel-jest": "^27.0.6",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-jest": "^24.3.5",
-    "expo": "^41.0.0",
+    "expo": "^42.0.0",
     "expo-app-loading": "^1.0.1",
-    "expo-asset": "~8.3.1",
-    "expo-av": "~9.1.2",
-    "expo-camera": "~11.0.2",
-    "expo-file-system": "~11.0.2",
-    "expo-firebase-analytics": "~4.0.2",
-    "expo-font": "~9.1.0",
-    "expo-image-manipulator": "~9.1.0",
-    "expo-image-picker": "~10.1.3",
-    "expo-linear-gradient": "~9.1.0",
-    "expo-screen-orientation": "~3.1.0",
+    "expo-application": "~3.2.0",
+    "expo-asset": "~8.3.2",
+    "expo-av": "~9.2.3",
+    "expo-camera": "~11.2.1",
+    "expo-constants": "~11.0.1",
+    "expo-device": "~3.3.0",
+    "expo-file-system": "~11.1.3",
+    "expo-firebase-analytics": "~4.1.0",
+    "expo-font": "~9.2.1",
+    "expo-image-manipulator": "~9.2.2",
+    "expo-image-picker": "~10.2.2",
+    "expo-linear-gradient": "~9.2.0",
+    "expo-notifications": "~0.12.2",
+    "expo-screen-orientation": "~3.2.1",
+    "expo-updates": "~0.8.0",
     "mime": "^2.5.2",
     "moment": "^2.29.1",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
     "react-native-draggable-flatlist": "^2.6.0",
-    "react-native-drax": "^0.7.2",
+    "react-native-drax": "latest",
     "react-native-gesture-handler": "~1.10.2",
     "react-native-markdown-display": "^7.0.0-alpha.2",
-    "react-native-reanimated": "~2.1.0",
+    "react-native-reanimated": "~2.2.0",
     "react-native-safe-area-context": "3.2.0",
-    "react-native-screens": "~3.0.0",
-    "react-native-svg": "12.1.0",
+    "react-native-screens": "~3.4.0",
+    "react-native-svg": "12.1.1",
     "react-native-svg-charts": "^5.4.0",
-    "react-native-tab-view": "^2.16.0",
+    "react-native-tab-view": "^3.0.1",
```

Comment:
nous oblige à installer react-native-pager-view depuis leur dernière version :
https://github.com/satya164/react-native-tab-view
https://github.com/satya164/react-native-tab-view/releases/tag/v3.0.0

Car react-native-gesture-handler et react-native-reanimated ne sont plus utilisé par le package. Je me suis noté dans le prochain ticket npm update mobile de creuser pourquoi et de creuser la lib react-native-pager-view.